linker/Tests/Mono.Linker.Tests.Cases/TypeForwarding/Dependencies/ForwarderLibrary.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using Mono.Linker.Tests.Cases.Expectations.Metadata;
+
+#if INCLUDE_FORWARDERS
+
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo (typeof (Mono.Linker.Tests.Cases.TypeForwarding.Dependencies.ImplementationLibrary))]
+
+#endif
+
+#if !INCLUDE_FORWARDERS
+namespace Mono.Linker.Tests.Cases.TypeForwarding.Dependencies {
+	// Need something here to keep the testing infrastructure happy
+	[NotATestCase]
```

Comment:
I'll look into it.  It's a bit of trading one bit of magic for another bit of magic :smile: